docker/Dockerfile.wifi-connect

```diff
@@ -0,0 +1,38 @@
+FROM screenly/srly-ose-base as builder
+
+WORKDIR /usr/src/app
+
+RUN curl https://api.github.com/repos/balena-io/wifi-connect/releases/latest -sL \
+    | grep -hoP 'browser_download_url": "\K.*armv7hf\.tar\.gz' \
+    | xargs -n1 curl -Ls \
+    | tar -xvz -C /usr/src/app/
+
+FROM balenalib/raspberrypi3:buster
```

Comment:
Shouldn't this be `FROM screenly/srly-ose-base as builder` too?

Reply:
Will use `screenly/srly-ose-base` as the base image. Thanks.